docs/plans/2026-07-16-pr-preview-unit1.md

```
@@ -0,0 +1,599 @@
+- Consumes: nothing.
+- Produces: a `gh-pages` branch whose root is the built site and which contains `.nojekyll`; Pages serving `gh-pages`/root. Task 2 relies on `gh-pages` existing and on `pr-preview/` being preserved by this workflow.
+
+- [ ] **Step 1: Generate and un-ignore the lockfile**
+
+```bash
+ws exec ken-site bundle install
+```
+
+Edit `components/ken-site/.gitignore` — delete the `Gemfile.lock` line so the Bundler block reads exactly:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)